plugins/org.eclipse.thym.core/src/org/eclipse/thym/core/config/WidgetModel.java

```diff
@@ -439,6 +451,63 @@ public void modelDirtyStateChanged(IStructuredModel model, boolean isDirty) {
 		}
 	}
 
+	private void updateEnginesFromConfigXML() {
```

Comment:
A good point. Moved this method to HybridMobileEngineManager, and added a convenience method to HybridProject. (Method has been renamed resyncWithConfigXml())